sim/sim.go

```diff
@@ -33,14 +33,18 @@ func NewSimulation(simConfig Config, graniteConfig f3.GraniteConfig, traceLevel
 	// Create a network to deliver messages.
 	lat := NewLogNormal(simConfig.LatencySeed, simConfig.LatencyMean)
 	ntwk := NewNetwork(lat, traceLevel)
-	vrf := f3.NewVRF(ntwk, ntwk)
+	if os.Getenv("F3_TEST_USE_BLS") == "1" {
+		ntwk.Signer = blssig.SignerWithKeyOnG2()
+		ntwk.Verifier = blssig.VerifierWithKeyOnG2()
+	}
```

Comment:
Since there's already a Config, I would prefer to use that than reach out to the OS from deeper within the code. The env var is a good idea, but perhaps fetch it from the test code rather than here (so that some tests can configure explicitly regardless of the environment).

Reply:
I left it still deep but allowed tests to override.